### PR TITLE
[luci/export] Revise encodeOpBufferByDType for extended Buffer

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise encodeOpBufferByDType for extended Buffer mode.
- store buffer content to extra map instead of creating Buffer
- return fake Buffer with offset/size of 1 and 1.

### CODE REVIEW

**Comment by seanshpark** [comment]
- for #13678
- from #13615

File: compiler/luci/export/src/CircleTensorExporter.cpp [hunk 1]
```diff
@@ -358,6 +358,21 @@ encodeOpBufferByDType(FlatBufferBuilder &builder, SerializedModelData &, luci::C
...
+    // TODO optimize this if this operation takes long or much memory
+    SerializedModelData::BufferData buffer_data;
+    buffer_data.resize(raw_size);
+    std::memcpy(buffer_data.data(), raw_data.data(), raw_size);
+
+    int32_t buffer_index = md._buffers.size();
+    md._buffer_data_map.emplace(buffer_index, buffer_data);
+
+    // create fake indicator buffer
+    return circle::CreateBuffer(builder, 0 /* data */, 1 /* offset */, 1 /* size */);
```
Comment (shs-park) [inline compiler/luci/export/src/CircleTensorExporter.cpp:373]: What is the purpose of this fake indicator buffer?
Is it used elsewhere? 😅

**Review by shs-park — APPROVED** [review]
I left a question, PTAL
=)

File: compiler/luci/export/src/CircleTensorExporter.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/export/src/CircleTensorExporter.cpp:373]: real (!fake) Buffer holds actual data,
but this fake Buffer won,t and the offset will point to the file offset to the actual Data.

**Pull request merged**